src/software/jetson_nano/services/motor.h

```diff
@@ -123,6 +124,72 @@ class MotorService
         }
     };
 
+    /**
+     * Log the driver fault in a human readable log msg
+     *
+     * @param motor The motor to log the status for
+     *
+     * @return a struct containing the motor faults and whether the motor was disabled due
+     * to the fault
+     */
+    struct MotorFaultIndicator checkDriverFault(uint8_t motor);
+
+    /**
+     * Wrapper function that writes int to the TMC4671
+     * @param motor drive motor number
+     * @param address motor address
+     * @param value write value
+     */
+    void writeIntToTMC4671(uint8_t motor, uint8_t address, int32_t value);
```

Comment:
can we added a comment that says "used for testing purpose" so that no one uses these in the future